src/gui/systray.h

```diff
@@ -89,7 +90,7 @@ public slots:
 
     bool _isOpen = false;
     bool _syncIsPaused = false;
-    QQmlApplicationEngine *_trayEngine;
+    QQmlApplicationEngine *_trayEngine = nullptr;
```

Comment:
I'd just use a QPointer here, just to be on the safe side (if destruction order somehow changes we could end up with a dangling pointer here).